src/main/java/moriyashiine/aylyth/common/item/AylythItems.java

```diff
@@ -172,6 +181,7 @@ public interface AylythItems {
     Item VITAL_THURIBLE = registerBlockItem("vital_thurible", AylythBlocks.VITAL_THURIBLE);
     Item BLACK_WELL = registerBlockItem("black_well", AylythBlocks.BLACK_WELL);
 
+    // TODO use simple numbers
```

Comment:
What

Reply:
I mean convert hex to decimal number, I'll improve the comment

Reply:
No, but why. That's significantly uglier and there is absolutely zero reason to reduce readability for no reason